FirebaseStorage/Sources/AsyncAwait.swift

```diff
@@ -110,6 +110,44 @@ public extension StorageReference {
     }
   }
 
+  /// Asynchronously uploads from a FileHandle to the currently specified StorageReference.
+  ///
+  /// - Parameters:
+  ///   - fileHandle: A file handle to the data to upload.
+  ///   - metadata: Optional StorageMetadata containing additional information (MIME type, etc.)
+  ///              about the object being uploaded.
+  ///   - onProgress: An optional closure function to return a `Progress` instance while the
+  /// upload proceeds.
+  /// - Throws: An error if the operation failed.
+  /// - Returns: `StorageMetadata` with additional information about the object being uploaded.
+  func putFileHandleAsync(_ fileHandle: FileHandle,
```

Comment:
Please add a test of this to FirebaseStorage/Tests/Integration/StorageAsyncAwait.swift.